.github/ISSUE_TEMPLATE/~good-first-issue.md

```diff
@@ -0,0 +1,14 @@
+---
+name: (Maintainers Only) Good First Issue
+about: For maintainers, to create an issue that is good for new contributors
+
```

Comment:
From step 5 [here](https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository) it looks like we can also add a `labels` key to have labels added automatically.